src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -39,6 +44,10 @@ const BottomSheetBio = React.forwardRef((props, ref) => {
   );
 });
 
+BottomSheetBio.propTypes = {
+  handleSave: PropTypes.func
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good to see PropTypes being used for type checking in React. However, only one prop `handleSave` is checked here. It would be better to include all props used in this component for comprehensive type checking.
```diff
  BottomSheetBio.propTypes = {
+   value: PropTypes.string,
+   error: PropTypes.string,
+   isOtherProfile: PropTypes.bool,
+   isLoadingUpdateBio: PropTypes.bool,
    handleSave: PropTypes.func
  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->